service/src/main/resources/application.yml

```
@@ -136,7 +135,7 @@ spring.config.activate.on-profile: prod
 drshub:
   compactIdHosts:
     "dg.4503": gen3.biodatacatalyst.nhlbi.nih.gov # biodatacatalyst
-    "dg.anv0": gen3.theanvil.io # old anvil
+    "dg.anv0": data.terra.bio # old anvil (now hosted on TDR)
```

Really minor but comments like `now hosted` only make sense relative to the time when the commit occurred. I would leave out the `now`.
```suggestion
    "dg.anv0": data.terra.bio # old anvil (hosted on TDR)
```

---

fair enough